### PR TITLE
Feature/cw 104 reference cases overview and detail page

### PR DESCRIPTION
**Pull request by Semperdecus**

### CODE REVIEW

File: src/components/cms-components/WorkCardList.astro [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import CasePreview from "../uielements/CasePreview.astro";
+
+interface Props {
+  workCardList: WorkCardListReferenced;
+}
+
+const { workCardList } = Astro.props;
+---
+
+<div>
```
Comment (DavithkbY) [inline src/components/cms-components/WorkCardList.astro:12]: Why two divs here? Is 1 not enough?

File: src/components/uielements/CasePreview.astro [hunk 2]
```diff
@@ -1,133 +1,111 @@
...
             class="caseheadline mb-2 font-serif text-xl [&>mark]:bg-none [&>mark]:bg-clip-border [&>mark]:text-black"
-            set:html={caseData.title}
+            set:html={caseData?.intro?.title}
           />
         </div>
         {
-          caseData.logo && (
-            <Image
+          caseData?.metadata?.image && (
+            <SanityImg
```
Comment (DavithkbY) [inline src/components/uielements/CasePreview.astro:58]: if you're creating a SanityImage component, then this component can also fetch the alt tags always no? I think the alt should be resolved inside the component, together with the src.

Comment (DavithkbY) [reply src/components/uielements/CasePreview.astro:58]: You should just pass i18nimage, classes, width/height, and loading to the SanityImage component and it should handle the rest.

File: src/components/uielements/CasePreview.astro [hunk 3]
```diff
@@ -1,133 +1,111 @@
...
+        caseData.intro?.intro && (
+          <div class="text-sm font-light [&>mark]:text-black">
+            <PortableText value={caseData.intro?.intro} />
+          </div>
+        )
+      }
       <ol class="mt-2 list-inside list-disc font-serif text-sm">
         {
-          caseData.duration && (
+          caseData.duration != null && (
```
Comment (DavithkbY) [inline src/components/uielements/CasePreview.astro:77]: you can just do caseData.duration && , I think when you do != null, you don't check for undefined.

File: src/components/uielements/CasePreview.astro [hunk 4]
```diff
@@ -1,133 +1,111 @@
...
               {t("metadata.duration") + ": " + caseData.duration}
             </li>
           )
         }
         {
-          caseData.collaboration && (
+          caseData.collaborationModel && (
             <li class="">
-              {t("metadata.collaboration") + ": " + caseData.collaboration}
+              {t("metadata.collaboration") + ": " + caseData.collaborationModel.collaborationTabs.map(tab => tab?.concept?.prefLabel).join(', ')}
```
Comment (DavithkbY) [inline src/components/uielements/CasePreview.astro:86]: this should probably be caseData?.collaborationModel?.collaborationTabs?.map just to be safe and that it won't crash in live edit mode.

File: src/components/uielements/CasePreview.astro [hunk 5]
```diff
@@ -1,133 +1,111 @@
...
             </li>
           )
         }
       </ol>
     </div>
     <Button
       class="self-end"
-      link={link}
-      style="primary"
+      link={`/preview/en/cases/${caseData?.metadata?.slug?.current}`}
```
Comment (DavithkbY) [inline src/components/uielements/CasePreview.astro:108]: use the isPreview variable here and also the siteLanguage variable like in other files to make it dynamic.

File: src/pages/preview/[lang]/cases/[slug].astro [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+import HeroHeader from "@components/uicomposition/HeroHeader.astro";
+import Customers from "@components/uicomposition/Customers.astro";
+import { isPreview, siteLanguage } from "src/stores/global-store";
+import type { SiteConfigQueryResult } from "@sanity/sanity.types";
+import type {WorkWithReferences} from "@sanity/custom.sanity.types";
+import ProjectMetadata from "../../../../components/uicomposition/ProjectMetadata.astro";
+import TechnologyIcon from "../../../../components/uielements/TechnologyIcon.astro";
+import SanityImg from "@otterstack/sanity-img-astro";
+import {getHrefFromLinkObject} from "../../../../lib/sanityHelpers";
+import CasePreview from "../../../../components/uielements/CasePreview.astro";
```
Comment (DavithkbY) [inline src/pages/preview/[lang]/cases/[slug].astro:15]: please remove all the relative imports. Use @components/... instead

**Review by DavithkbY — CHANGES_REQUESTED** [review]

File: src/components/uielements/CasePreview.astro (hunk 2 above)
Comment (Semperdecus) [reply src/components/uielements/CasePreview.astro:58]: Doesn't seem to be the case, without the alt tag I don't see an alt tag. This is what you mean right?
```
            <SanityImg
              loading="lazy"
              src={caseData?.metadata?.image}
              class="h-12 w-auto shrink-0 sm:h-16 md:h-12 xl:h-16"
              height={60}
            />
```

**Pull request merged**